youtube_dl/extractor/vvvvid.py

```diff
@@ -156,3 +165,56 @@ def f(m):
             'view_count': int_or_none(video_data.get('views')),
             'like_count': int_or_none(video_data.get('video_likes')),
         }
+
+
+class VVVVIDShowIE(VVVVIDIE):
+    _VALID_URL = r'(?P<base_url>%s(?P<show_id>\d+)/(?P<id>[^/]+))/?(?:$|[\?&].*$)?$' % VVVVIDIE._VALID_URL_BASE
+    _TESTS = [{
+        'url': 'https://www.vvvvid.it/show/156/psyco-pass',
+        'info_dict': {
+            'id': '156',
+            'title': 'Psycho-Pass',
+            'description': 'md5:94d572c0bd85894b193b8aebc9a3a806',
+        },
+        'playlist_count': 46,
+    }]
+
+    def _real_extract(self, url):
+        base_url, show_id, video_id = re.match(self._VALID_URL, url).groups()
+
+        response = self._download_info(
+            '%s/seasons/' % show_id, video_id)
+
+        show_infos = self._download_info(
+            '%s/info/' % show_id, video_id)
+
+        show_title = show_description = None
+        if show_infos:
+            show_title = show_infos['data'].get('title')
+            show_description = show_infos['data'].get('description')
+
+        entries = []
+        for season in response['data']:
+            if season['episodes']:
+                for episode in season['episodes']:
+                    season_id = str_or_none(episode.get('season_id'))
+                    video_id = str_or_none(episode.get('video_id'))
+                    if not (season_id and video_id):
+                        continue
+
+                    video_url = '/'.join([base_url, season_id, video_id])
+
+                    entries.append({
+                        '_type': 'url_transparent',
+                        'ie_key': VVVVIDIE.ie_key(),
+                        'url': video_url,
+                        'title': episode.get('title'),
+                        'thumbnail': episode.get('thumbnail'),
+                        'description': episode.get('description'),
+                        'season_number': int_or_none(episode.get('season_number')),
+                        'episode_number': int_or_none(episode.get('number')),
+                    })
+        return self.playlist_result(entries,
+                                    playlist_id=show_id,
+                                    playlist_title=show_title,
+                                    playlist_description=show_description)
```

Comment:
when you're using the kwargs in order, you don't have to use the kwarg name.
you can just do(after moving `data` extraction to `_download_info`):
```suggestion
        return self.playlist_result(
            entries, show_id, show_info.get('title'), show_info.get('description'))
```